modifiers/modifiers.go

```diff
@@ -16,6 +16,8 @@ func New() *mold.Transformer {
 	mod.Register("lcase", ToLower)
 	mod.Register("ucase", ToUpper)
 	mod.Register("snake", SnakeCase)
+	mod.Register("num", NumCase)
+	mod.Register("notnum", NotNumCase)
```

Comment:
Not in love with the tag and function names, suggestion:
- strip_alpha
- strip_num

Reply:
I agree. Done.